text_mining_tfidf.py

```diff
@@ -0,0 +1,236 @@
+"""Mines text from Project Gutenburg. Performs TF-IDF and sentiment analysis
+on text and generates a word cloud from most important words.
+
+@author: Vivien Chen
+
+"""
+
+import requests
+import string
+import math
+from pickle import dump, load
+from os.path import exists
+from stop_words import get_stop_words       # pip install stop_words
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from wordcloud import WordCloud             # pip install wordcloud
+from textblob import TextBlob as tb         # pip install textblob
+
+
+def get_cache(url, file_name):
```

Comment:
One thing to look into: you can import your own functions in Python (e. g., "from text_mining import get_cache" - it'd save you some repeated code!